plugins/SegmentEditor/templates/_segmentSelector.twig

```diff
@@ -43,8 +43,8 @@
 
                 {{ 'SegmentEditor_ThisSegmentIsVisibleTo'|translate }} <div class="enable_all_users"><strong>
                         <select class="enable_all_users_select">
-                            <option value="0">{{ 'SegmentEditor_VisibleToMe'|translate }}</option>
-                            <option selected="1" value="1">{{ 'SegmentEditor_VisibleToAllUsers'|translate }}</option>
+                            <option {% if not isSuperUser %}selected="1"{% endif %} value="0">{{ 'SegmentEditor_VisibleToMe'|translate }}</option>
```

Comment:
was this change meant @diosmosis ? Seems maybe unrelated to this PR?

Reply:
I think what was meant to happen is that "preprocessed reports" should be selected by default when possible (eg when browser archiving is disabled aka a cron was set up).

"Preprocessed reports" should also only be selectable ideally when browser archiving is disabled.

Reply:
This is to allow view access users to be able to create segments, otherwise the creation will always fail because the default is "visible to all users".

Reply:
👍